core/iwasm/fast-jit/jit_ir.h

```diff
@@ -313,7 +313,8 @@ typedef struct JitInsn {
     /* Opcode of the instruction. */
     uint16 opcode;
 
-    /* Reserved field that may be used by optimizations locally. */
+    /* Reserved field that may be used by optimizations locally.
+     * LSB is atomic flag for load/store */
```

Comment:
bit_0 is atomic flag for load/store

We may use bit_1 as GS register flag in the future.